### PR TITLE
ss-135: Add validation messages for "Sign Up" page

### PR DESCRIPTION
**Pull request by teslmik**

### CODE REVIEW

File: frontend/src/bundles/auth/pages/auth.tsx [hunk 1]
```diff
@@ -59,19 +59,24 @@ const Auth: React.FC = () => {
...
+                                <div className={styles.headerText}>
+                                    <span>
+                                        {getText(pathname, 'headerText')}
+                                    </span>
+                                    <Link
+                                        className={styles.headerLink}
+                                        to={
+                                            pathname === AppRoute.SIGN_IN
+                                                ? AuthApiPath.SIGN_UP
+                                                : AuthApiPath.SIGN_IN
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/auth/pages/auth.tsx:74]: It's better to create a variable than to write the whole condition in the layout

File: frontend/src/bundles/common/helpers/get-text-to-path/get-text-to-path.ts [hunk 2]
```diff
@@ -8,9 +8,14 @@ const getText = (pathname: string, type: string): string => {
         [AppRoute.SIGN_UP]: 'Sign Up',
     };
 
+    const headerText: Propertys = {
```
Comment (VItaliy-Kalenichenko) [inline frontend/src/bundles/common/helpers/get-text-to-path/get-text-to-path.ts:11]: Try to come up with a better name for this variable

File: shared/src/bundles/users/validation-schemas/user-sign-up.validation-schema.ts [hunk 3]
```diff
@@ -6,19 +6,34 @@ import { type UserSignUpRequestDto } from '../types/types.js';
 const userSignUp = joi.object<UserSignUpRequestDto, true>({
     email: joi
         .string()
-        .trim()
+        .pattern(
+            new RegExp(
+                '^[a-zA-Z0-9._%+-]{1,15}@(?=.{3,15}$)[a-zA-Z0-9.-]{1,}\\.[a-zA-Z]{1,}$',
```
Comment (VItaliy-Kalenichenko) [inline shared/src/bundles/users/validation-schemas/user-sign-up.validation-schema.ts:11]: What is this for?
Even if you need it, it's better to assign regular expressions to a variable

File: frontend/src/assets/css/variables/padding-variables.scss [hunk 4]
```diff
@@ -7,4 +7,5 @@
     --p-6: 3.5rem;
     --p-7: 7.5rem;
     --p-8: 8.5rem;
+    --p-9: 2.5rem;
```
Comment (nikita-remeslov) [inline frontend/src/assets/css/variables/padding-variables.scss:10]: I am not sure that you need this padding.

**Review by nikita-remeslov — APPROVED** [review]

**Comment by nikita-remeslov** [comment]
Conflicts

**Review by nikita-remeslov — APPROVED** [review]

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**